harvester/records/formats/marc.py

```diff
@@ -75,8 +75,8 @@ def _gbl_resourceClass_sm(self) -> list[str]:
             "cartographic three-dimensional form": "Other",
         }
 
-        tag_336_value = self.get_single_tag_subfield_value("336", "a")
-        return [tag_336_to_aardvark_map[tag_336_value]]
+        tag_336_values = self.get_multiple_tag_subfield_values([("336", "a")])
+        return [tag_336_to_aardvark_map[value] for value in tag_336_values]
```

Comment:
[Non-blocking] Do the keys in`tag_336_to_aardvark_map` represent the complete set from MIT ALMA GIS MARC records? How were these values derived?

Reply:
Also, why does `cartographic image` map to "Imagery" but the plural form `cartographic images` maps to "Maps"? 🤔

Reply:
> [Non-blocking] Do the keys in`tag_336_to_aardvark_map` represent the complete set from MIT ALMA GIS MARC records? How were these values derived?

Yes!  Taking all ~7k records, then analyzing unique values for tag `336$a` resulted in this pretty tightly controlled set here.  Here is the breakdown:

```
+-----------------------------------+--------+
|a                                  |count(a)|
+-----------------------------------+--------+
|cartographic image                 |5975    |
|text                               |255     |
|cartographic dataset               |74      |
|unspecified                        |7       |
|still image                        |5       |
|cartographic images                |1       |
|computer dataset                   |1       |
|cartographic three-dimensional form|1       |
+-----------------------------------+--------+
```

These values all have 2nd indicator as `rdacontent`, and map to this list of controlled terms: https://www.loc.gov/standards/valuelist/rdacontent.html.  So, theoretically we could proactively map all these. 
 Unsure if that's needed at this time or a nice improvement later.

Although this brings up a question I struggled with.  This field helps distinguish between visual materials and data, but we might benefit from looking at `007`, `255`, and `300` fields to distinguish between "maps" and "images".

Going to dig into this a bit more, thanks for prompting some reflection here.

> Also, why does `cartographic image` map to "Imagery" but the plural form `cartographic images` maps to "Maps"? 🤔

Good catch!  This may have just been an error.

Reply:
Just typed a long response, but lost it, ergh!  But in short...

> [Non-blocking] Do the keys in`tag_336_to_aardvark_map` represent the complete set from MIT ALMA GIS MARC records? How were these values derived?

Yep!  Analyzed `336$a` and found this fairly tightly controlled set:

```
+-----------------------------------+--------+
|a                                  |count(a)|
+-----------------------------------+--------+
|cartographic image                 |5975    |
|text                               |255     |
|cartographic dataset               |74      |
|unspecified                        |7       |
|still image                        |5       |
|cartographic three-dimensional form|1       |
|cartographic images                |1       |
|computer dataset                   |1       |
+-----------------------------------+--------+
```

These tags all have 2nd indicator of `rdacontent` and therefore are controlled terms from this list: https://www.loc.gov/standards/valuelist/rdacontent.html.  So, in theory we could map others we haven't seen yet.

That said, while this helps distinguish between visual resources and data, it doesn't help much with "maps" vs "images" (relevant to your other question).

I think we may be able to get some clues from fields `007, 255, 300` that might help with that.  Going to expand this logic a bit as a new commit.

Reply:
Looking again, believe we can use tag `336` (like before) to understand Dataset vs Image vs Other, and then can use tag `007` character `00` to understand if Map or Image.

With that logic in place, here is the breakdown:

```
┌──────────────────────┬─────────────────────────────┐
│ gbl_resourceClass_sm │ count(gbl_resourceClass_sm) │
│      varchar[]       │            int64            │
├──────────────────────┼─────────────────────────────┤
│ [Maps]               │                        5486 │
│ [Imagery]            │                         262 │
│ [Maps, Other]        │                         214 │
│ [Datasets]           │                          73 │
│ [Other]              │                          33 │
│ [Other, Imagery]     │                          11 │
│ [Imagery, Other]     │                           3 │
│ [Other, Maps]        │                           2 │
│ [Maps, Datasets]     │                           1 │
└──────────────────────┴─────────────────────────────┘
```

The legacy system used the value `"Paper Map"` for everything, but that is not an allowed term in Aardvark, and unsure it's accurate.  Feeling better about this slightly enhanced logic in this commit.

Reply:
Thank you for looking into this and providing more context! I agree with the updates made.